lib/rspec/matchers/built_in/raise_error.rb

```diff
@@ -148,14 +152,22 @@ def warning_about_bare_error
         end
 
         def warn_about_bare_error
-          RSpec.warning("Using the `raise_error` matcher without providing a specific " \
-                        "error or message risks false positives, since `raise_error` " \
-                        "will match when Ruby raises a `NoMethodError`, `NameError` or " \
-                        "`ArgumentError`, potentially allowing the expectation to pass " \
-                        "without even executing the method you are intending to call. " \
-                        "Instead consider providing a specific error class or message. " \
-                        "This message can be supressed by setting: " \
-                        "`RSpec::Expectations.configuration.warn_about_potential_false_positives = false`")
+          warn_about(
+            "Using the `raise_error` matcher without providing a specific " \
+            "error or message risks false positives, since `raise_error` " \
+            "will match when Ruby raises a `NoMethodError`, `NameError` or `ArgumentError`. ",
+            "Instead consider providing a specific error class or message. "
+          )
+        end
+
+        def warn_about expression, alternative
+          warning = "This will potentially allowing the expectation to pass without" \
+                    "without even executing the method you are intending to call. "
+
+          instructions = "This message can be supressed by setting: " \
+                         "`RSpec::Expectations.configuration.warn_about_potential_false_positives = false`"
+
+          RSpec.warning(expression + warning + alternative + instructions)
```

Comment:
Personally, I think it's incidental duplication: just because there is some common text now, doesn't mean there will always be.  We may want to reword one message and the common helper method would probably get in the way.  Plus it's harder to get the sense for what the entire message will be when its broken up like this.

Do you mind defactoring it inspite of the textual duplication?